README.md

```diff
@@ -1 +1,4 @@
-The taxi for a little city
\ No newline at end of file
+The taxi for a little city
+###Contributions:
+Yelizaveta
```

Comment:
Please write your name and your email on the same line :construction_worker_man: